profile.js

```diff
@@ -1,10 +1,20 @@
 const userNameEle = document.getElementById("userName");
+const navigateEle = document.getElementById("navigate-user-text");
+
+// Navigate to practice.html if no data in local storage
+const checkIfUserData = () => {
+    const name = localStorage.getItem("typerName");
+    if(!name){
+        navigateEle.innerHTML = "<a href='./practice.html'>Click here</a> to set up your profile";
```

Comment:
Instead of using `innerHTML` use `Element.insertAdjacentElement()` function to add element. 

To learn more about `Element.insertAdjacentElement()` visit https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentElement

Reply:
![image](https://user-images.githubusercontent.com/58901210/197480270-c1c1d8f4-c844-4f92-b0fd-b617a8d5c086.png)
Is this code looks good, Can I make a commit arpitghura? 
                                         (or)
do you also want me to remove the empty p tag in the profile.html

Reply:
yeah, the code looks also do increase the font size of the text.

Reply:
![image](https://user-images.githubusercontent.com/58901210/197758303-e00f638c-a059-410a-b5a1-7d068d2eafc5.png)

Reply:
What about 20px?

Reply:
![image](https://user-images.githubusercontent.com/58901210/197760409-f2549053-d96e-4c15-bb55-2c7a256ff633.png)

Reply:
Good?

Reply:
Yes, this is looking great 😍